packages/phone-number-privacy/package.json

```diff
@@ -18,7 +18,7 @@
     "db:migrate": "NODE_ENV=development ts-node scripts/runMigrations.ts"
   },
   "dependencies": {
-    "@celo/contractkit": "0.3.2",
+    "@celo/contractkit": "0.3.9-dev",
```

Comment:
we can only used published versions in firebase, so we may have to wait if your contractkit changes aren't published yet

Reply:
Thanks! I'll split this into two separate PRs then. Once the contractkit can be published then I'll update this version number.